xml_inscriptions/bib_only/MD.Balt.BMA.G.1940.164.xml

```diff
@@ -25,15 +25,15 @@
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
                   <msItem class="#funerary.epitaph">
-                     <p>Funerary stele</p>
+                     <p>funerary stele</p>
                   </msItem>
                </msContents>
                <physDesc>
 
                   <!--  *************************************  <objectDesc>  *********************************  -->
 
                   <objectDesc ana="#stele">
-                     <supportDesc ana="#stone.limestone">
+                     <supportDesc ana="#stone.other">
                         <support>
                            <p/>
```

Comment:
Include the information about type of stone in this `<p>` element; this seems to be something you'll need to do throughout. Here, you'll put "limestone stele".